.github/release_issue_template.md

```diff
@@ -0,0 +1,20 @@
+# Documentation Release Checklist
+
+Use this issue template for documentation releases.
+
+## Packaging Day:
+
+- [ ] Ensure all relevant PRs are merged (and backported where required)
+- [ ] Cut a new branch from master using syntax `manager-x.y-betaz`
+- [ ] Build both SUMA and Uyuni docs from release branch and visually check output
```

Comment:
```suggestion
- [ ] Build both SUMA and Uyuni docs from release branch and visually check output: https://github.com/uyuni-project/uyuni-docs/wiki/building-documentation
```
Or do we have a better recipe?

Reply:
i was thinking more `make antora-suma` and `make anotra-uyuni` ;)